lib/ansible/modules/identity/keycloak/keycloak_client_scope.py

```diff
@@ -0,0 +1,473 @@
+        default: False
+
+    target:
+        description:
+            - Scope mappings can be acted upon for either clients or client mappings. Choose one
+              here; the default ist C(client).
+        default: 'client'
+        choices: ['client', 'client-template']
+
+    id:
+        description:
+            - id of client or client template to work on scope mappings on. Either this or one of
+              I(client_id)/I(name) are required.
+
+    name:
+        description:
+            - name of client template to work on (if I(target) is C('clienttemplate')). Either this
+              or I(id) is required.
+
+    realm:
+        description:
+            - realm this scope mapping belongs to.
+
+    client_id:
+        description:
+            - client_id of client to work on (if I(target) is C('client')). Either this or I(id)
+              is required.
+
+    type:
+        description:
+            - Type of scope mapping.
+            - On C(realm), the scope mappings are for realm scopes. (In this case C(roles) is
+              required)
+            - On C(client), the scope mappings are for client scopes. (In this case C(clientroles)
+              is required)
+        choices: ['realm', 'client']
+        default: 'realm'
+
+    roles:
+        description:
+            - A list of role representations (in dict-form) to be acted on. Usually, only C(name) is
+              required. Either C(roles) or C(clientroles) is required, depending on C(type).
+              U(http://www.keycloak.org/docs-api/3.3/rest-api/index.html#_rolerepresentation) can be
+              referenced for the structure. Roles need to exist already to be put in scopes.
+        suboptions:
+            name:
+                description:
+                    - Name of the role
+                required: true
+
+            clientRole:
+                description:
+                    - Boolean specifying whether this is a clientRole
+
+            composite:
+                description:
+                    - Boolean specifying whether this is a composite role. If so, I(composites)
+                      contains its definition.
+
+            composites:
+                description:
+                    - if I(composite) is set to True, this defines the composite role definition through
+                      a dict. The set of all the rolenames specified therein is the set of roles this
+                      composite role is comprised of. The dict contains the keys I(client) and I(realm).
+                    - I(client) contains another dict with client names as keys and lists of client
+                      role names as values.
+                    - I(realm) contains a list of realm role names.
+
+            containerId:
+                description:
+                    - Container id of this role - usually the realmname.
+
+            description:
+                description:
+                    - Description of this role.
```

Comment:
The description of the role is not used when linking the role into the scope and this does not update the role description. Can the documentation about unused feature can be deleted?